### PR TITLE
feat: add support for sepolia

### PR DESCRIPTION
**Pull request by superical**
## Summary
Add support for sepolia testnet.

## Changes
- Add sepolia network information to config file

## Issues
- https://www.pivotaltracker.com/story/show/183149344

### CODE REVIEW

File: src/commands/networks.ts [hunk 1]
```diff
@@ -48,6 +49,10 @@ export const supportedNetwork: {
     explorer: "https://goerli.etherscan.io",
     provider: defaultInfuraProvider("goerli"),
   },
+  [NetworkCmdName.Sepolia]: {
+    explorer: "https://sepolia.etherscan.io",
+    provider: jsonRpcProvider("https://rpc.sepolia.org"),
+  },
```
Comment (simboonlong) [inline src/commands/networks.ts:55]: what's the purpose of sepolia? i thought goerli is enough 🤔

anyway i tried sepolia out, `deploy` doc store and `issue` works fine but the `verify` command `npm run dev -- verify --document <YOUR_DOCUMENT> --network sepolia` and it throws:

```
Error: unsupported network (argument="network", value=null, code=INVALID_ARGUMENT, version=providers/5.4.5)
```
etherjs don't support `sepolia` it seems? ethers.js/issues/3325

Comment (superical) [reply src/commands/networks.ts:55]: > what's the purpose of sepolia? i thought goerli is enough

Just like we have ropsten and rinkeby? But these will be deprecated soon. 😢

> anyway i tried sepolia out, deploy doc store and issue works fine but the verify command npm run dev -- verify --document <YOUR_DOCUMENT> --network sepolia and it throws:

Ah, noice! Looks like we forgotten to update the provider in verification in #210.😅 Fixed it in https://github.com/Open-Attestation/open-attestation-cli/pull/215/commits/defbe6877df282b0a0725dcdae0524657a3c15f4. 

This PR is also pending the PR at https://github.com/Open-Attestation/dnsprove/pull/28 and https://github.com/Open-Attestation/dnsprove/pull/29 to get merge first and another one for the verify to resolve another verification issue.

> etherjs don't support sepolia it seems? ethers.js/issues/3325

Your PR in #210 has helped to support it and any other "exotic" networks for CLI.😃👍

Comment (superical) [reply src/commands/networks.ts:55]: @simboonlong Thanks for reviewing the PRs! I believe the verify should work now!

**Review by simboonlong — APPROVED** [review]
home run! 😂

**Pull request merged**